src/geo_traits_impl/multi_point.rs

```diff
@@ -0,0 +1,59 @@
+use super::{MultiPointType, PointType};
+use bytemuck::TransparentWrapper;
```

Comment:
What's the benefit of using this?

Reply:
I actually don't know another way of implementing `geo-traits` on the `geojson` types without it.

When we implement `GeometryTrait` for `geojson::Geometry` and `geojson::Value`, we need to specify all the associated types like `PointType`, but we can't implement `PointTrait` for `PointType` because `PointType` is just a `Vec<f64>` (which we can't implement traits for).

So we need to have a wrapper / newtype wrapping `geojson::PointType`, but `geo-traits` expects it to be a reference, so we have to swap out the `&Vec<f64>` with our newtype, which `bytemuck` can do.

Is there a cleaner way to do this without this new dependency?

Reply:
I understand why you have to do this the way things are currently defined.

I'm having trouble seeing how this will all tie together - can you include a test or example that shows parsing some geojson and then using the geo-traits functionality on the resultant object?

Reply:
Something like this? https://github.com/georust/geojson/pull/245/commits/466a5cbdcf266b933a2ac21459c18a031bad855d

Reply:
For the sake of saying another option, we could also introduce newtypes that wrap `Vec<f64>` so instead of `type Position = Vec<f64>` we have `struct Position(Vec<f64>)`, which I think allow us to not have to use `bytemuck`. But that's a much larger change (in terms of code diff, not conceptually).

Reply:
>> can you include a test or example 
>
> Something like this? 

Thank you for the example - it's better than I feared! So we have these new types (via bytemuck) but it seems like users don't really need to know about them? I don't really understand how this works TBH.

My initial thought, if we had to foist the mental burden of new types onto users, would be to bite the bullet and make them full citizens to fully replace all the existing type aliases.

That would also pave the way for #222, which is predicated on introducing a newtype for Position (probably with private members).

But with bytemuck, you've hit kind of an easy compromise. It's a bit of a local maxima, but not a bad one to be at.

If I had to bet, I would bet that "one day" geojson will have newtypes with private members rather than a bunch of type aliases for it's foundational data types, so I'd be in favor of ripping off that bandaid now if it's useful. But I'm also OK with the bytemuck approach for now if you prefer it.

Reply:
> but it seems like users don't really need to know about them?

Yeah it all happens behind the scenes when someone uses the `geo_traits` traits on the `GeoJson` types. They newtypes created in this file aren't even exposed publicly.